src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir-Book, sans-serif;
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+  font-size: 16px;
+}
+
+body {
+  margin: 20px 10px 0;
+}
+
+.form {
+  display: inline-block;
+  width: 100%;
+  margin-bottom: 20px;
+  background: #fff;
+  /* border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px; */
```

Comment:
let's remove this, there should be a border only on the input
box-shadow as well, only on the input